documentation/src/main/asciidoc/topics/ref_config_programmatic.adoc

```diff
@@ -0,0 +1,88 @@
+[id='programmatic-{context}']
+= Programmatic Configuration
+Create new Configuration objects with the `ConfigurationBuilder` class and then
+define cache configurations with the Cache Manager.
+
+[NOTE]
+====
+The examples in this section use `EmbeddedCacheManager`, which is a Cache
+Manager that runs in the same JVM as the client.
+
+To configure caches remotely with HotRod clients, you use `RemoteCacheManager`.
+Refer to the HotRod documentation for more information.
```

Comment:
Can we link to the HotRod documentation? Or does that not work with how things are separated?

Reply:
@ryanemerson we do link to the HotRod docs but from the `.References` section. Linking between guides in community is really straightforward. I've added attributes so all you need to do is `link:{hotrod_docs}[HotRod Java Clients]`. 

you can add anchors as well like `link:{hotrod_docs}#anchor_name[HotRod Java Clients]`.

also pushing links to a reference section keeps readers "on task". inline linking can lead to distractions. I could go all day with this... let me know what you think though. I can show you some examples and explain in more detail.